src/ruby.rs

```diff
@@ -197,13 +197,17 @@ impl zed::Extension for RubyExtension {
                 if let Some(script) = &ruby_config.script {
                     arguments.push(script.clone());
                 } else if let Some(command) = &ruby_config.command {
-                    arguments.push("--command".to_string());
-                    arguments.push(command.clone());
+                    arguments.extend(["--command".into(), "--".into(), command.clone()]);
                 } else if let Some(command_or_script) = &ruby_config.script_or_command {
                     if worktree.which(command_or_script).is_some() {
```

Comment:
*thought (non-blocking):* I wonder why we need to check if `command_or_script` exists in a worktree here. We should check that later.